src/token/PendingAirdropId.js

```diff
@@ -121,6 +122,7 @@ export default class PendingAirdropId {
      * @returns {this}
      */
     setNftId(nftId) {
+        this._tokenId = null;
```

Comment:
Isn't `_tokenId` null set in the constructor?

Reply:
PendingAirdropId should have either tokenId or nftid. So in case he sets nftId we should null the nftId and vice versa.

Reply:
nit: Would expect the setter to be with an individual responsibility. But in this specific case it is ok not to change it since it will trigger other changes for you.